javatests/com/williamfiset/datastructures/splaytree/SplayTreeTest.java

```diff
@@ -0,0 +1,55 @@
+package javatests.com.williamfiset.datastructures.splaytree;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class SplayTreeTest {
+
+    @Test
+    void getRoot() {
+        SplayTree<Integer> splayTree = new SplayTree<>();
+        assertNull(splayTree.getRoot());
+        splayTree.insert(2);
+        assertEquals(2,splayTree.getRoot().getData());
+        splayTree.insert(3);
+        assertEquals(3,splayTree.getRoot().getData());
+        splayTree.insert(10);
+        assertEquals(10,splayTree.getRoot().getData());
+
+    }
+
+    @Test
+    void testSplayInsertDeleteSearch() {
+        SplayTree<Integer> splayTree = new SplayTree<>();
+        int[] data = {2, 29, 26,-1,10,0,-2,11};
+        for (int i :
+                data) {
+            splayTree.insert(i);
+            assertEquals(i,splayTree.getRoot().getData());
+        }
+        for (int i :
+                data) {
+            splayTree.delete(i);
+            assertNull(splayTree.search(i));
+        }
+
+    }
+
+    @Test
+    void findMax() {
+        SplayTree<Integer> splayTree = new SplayTree<>();
+
+        int[] data = {2, 3, 4,5,6,7,8,9};
+        for (int i :
+                data) {
+            splayTree.insert(i);
+            assertEquals(i,splayTree.findMax(splayTree.getRoot()));
+        }
+
+    }
+
+
+
+
```

Comment:
What did you change in the ds_intro_slides.key, ds_intro_slides.pdf and  ds_intro_slides.pptx? Can you revert those changes?

Reply:
I honestly dont know. How something got changed there.I didnot change anything inside the slides folder on purpose. It might be due to some refactoring i made on other files :'
I'll overwrite my version of the slides folder with your original version and then commit.

Reply:
Yes sg. Please sync and remove the diff would be great.